src/main/java/uk/gov/hmcts/ccd/domain/service/createevent/AuthorisedCreateEventOperation.java

```diff
@@ -76,8 +76,7 @@ public CaseDetails createCaseEvent(String caseReference,
 
         verifyUpsertAccess(content.getEvent(), content.getData(), existingCaseDetails, caseTypeDefinition, userRoles);
 
-        final CaseDetails caseDetails = createEventOperation.createCaseEvent(caseReference,
-                                                                             content);
```

Comment:
Could you please take developed  version  and exclude this file from the pr? unless checkstyle crash :)

Reply:
reverted to develop version